<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hudview</class>
 <widget class="QWidget" name="hudview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>120</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QProgressBar" name="pb_hp">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>220</width>
     <height>25</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="format">
    <string notr="true">HP :%v/%m</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="pb_exp">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>220</width>
     <height>15</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="format">
    <string notr="true">Exp : %v/%m</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_name">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>130</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="l_level">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Level</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_lvl">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
